tasks/interactive-bots.md

```diff
@@ -59,14 +59,14 @@ Follow the tutorial below to test run the followup bot on your Zulip instance
   site=<dev-url>
   ```
 
-  In the example replace the placeholder `<api-key>` with an existing API
-  key and placeholder `<email>` with the bot's email. The placeholder 
-  `<dev-url>` should point to your development environment URL.
+-  In the example replace the placeholder `<api-key>` with an existing API
+-  key and placeholder `<email>` with the bot's email.The placeholder
```

Comment:
Why did you remove the whitespaces here? There should be a space before and after `The placeholder`.